pkgs/development/python-modules/dash-bootstrap-components/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dash
+}:
+
+buildPythonPackage rec {
+  pname = "dash-bootstrap-components";
+  version = "1.1.0";
+
+  buildInputs = [
+    dash
+  ];
```

Comment:
This needs to be propagatedBuildInputs.

Reply:
resolving this since it was posted twice.